src/library/GenerateNominations/useFetchMethods.tsx

```diff
@@ -129,7 +137,10 @@ export const useFetchMehods = () => {
   };
 
   const available = (nominations: any) => {
+    console.log(`Validators to be available for nominations:`);
+    console.log(nominations);
     const _nominations = Object.assign(validators);
+    console.log(_nominations);
```

Comment:
Why do we need it? Leaving `console.logs` usually considered as bad practice in frontend apps

Reply:
It's a mistake, probably didn't commit the file

Reply:
done